### ISSUE TITLE
Doesn't trigger as new chat messages come through

### ISSUE THREAD

**Issue by erichaus** [NekR/ResizeObserver-Intersect#2]
Hey so I tried this out in my chat app and as new messages come in, and the message-list target node expands, I thought ResizeObserver should fire(?) It only triggers once on the initial page render in this case... Am I off on how this should work?
```
    const rs = new ResizeObserver(() => {
      console.log(arguments);
    });
    rs.observe(document.querySelector('.message-list'));
```

**Comment by NekR** [comment]
@erichonkanen hey! this is an experimental polyfill, i for sure isn't meant or ready for production.

But thanks for experimenting with it, I'll take a look at the problem!

**Comment by erichaus** [comment]
hey hey, no problem! yeah im just experimenting after researching ways to keep an element scrolled to bottom e.g. chat. The current recommended way to do it `el.scrollTop = el.scrollHeight` isn't always reliable so this is interesting...

**Comment by cristianl** [comment]
I got resize detection working for my use case by giving the `resizer` element a width greater than 100% (may work for height too, I didn't try it yet).

Any value slightly over 100% (e.g. `calc(100% + 2px)`, `101%`) seems to give the same result.

@NekR Thanks for this polyfill! You [mentioned missed frames](https://github.com/NekR/ResizeObserver-Intersect/issues/1#issuecomment-338987610) and I did notice them, but only in Chrome/Blink (as part of testing, I removed the check for native ResizeObserver) so I believe [this is the cause](https://bugs.chromium.org/p/chromium/issues/detail?id=737228), but with native RO in Chrome 64+ it's less of a problem anyway.
